lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -103,38 +103,32 @@ impl Client {
     /// update_clipboard is invoked from Go.
     /// It updates the local clipboard cache and returns the encoded message
     /// that should be sent to the RDP server.
+    ///
+    /// The data passed to this function should be valid UTF-8 text.
     pub fn update_clipboard(&mut self, data: Vec<u8>) -> RdpResult<Vec<Vec<u8>>> {
-        const CR: u8 = 13;
-        const LF: u8 = 10;
-
-        // convert LF to CRLF, as required by CF_OEMTEXT
+        // convert LF to CRLF, as required by CF_TEXT and CF_UNICODETEXT
         let len_orig = data.len();
         let mut converted = Vec::with_capacity(len_orig);
         for i in 0..len_orig {
             match data[i] {
-                LF => {
```

Comment:
I felt this looked clearer but open to feedback.

Reply:
No objections from me.